src/components/utils.js

```diff
@@ -12,3 +12,12 @@ export function clearChildren(parent) {
 export function getNodeElementIndexFromNodeList(nodeElement, nodeList) {
   return Array.from(nodeList.children).findIndex(item => item.innerText === nodeElement.innerText);
 }
+
+export function displayZeroPropertiesFoundError(spinner) {
+  if (!state.properties.length) {
+    renderPage('search');
+    spinner.toogleVisibility(false);
+    renderError('No properties found in this area');
+    return;
+  }
+}
```

Comment:
I guess it'd better to move away 'if' from here, usually functions that display, render smth are supposed to be written without any conditions